PSOperations/RetryOperation.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RetryOperation.swift
+//  PSOperations
+//
+//  Created by Bart Whiteley on 9/22/15.
+//
+
+
+public class RetryOperation: GroupOperation {
+    private let operationProducer: () -> Operation
+    private let shouldRetry: (errors:[NSError], retryCount:Int) -> Bool
+    private var numberOfRetries: Int = 0
+    
+    public convenience init(@autoclosure(escaping) operation: () -> Operation, maximumNumberOfRetries: Int = 5) {
+        self.init(operationProducer: operation, shouldRetry: { (_, retries) in return retries < maximumNumberOfRetries })
```

Comment:
If I'm reading this right, it looks like this convenience `init` makes the `RetryOperation` work as more of a `RepeatOperation` where you specify how many times you want it to repeat. Is that correct?